unit-tests/game-api-tests.js

```diff
@@ -1,32 +1,23 @@
-//describe('gameApiTest', function(){
-//
-//    var gameApi;
-//
-//    beforeEach(function(){
-//        module('tombola.noughtAndCrosses.services');
-//        inject(function($injector){
-//            gameApi = $injector.get('gameApi');
-//        });
-//    });
-//
-//    it('Should contain gameApi service', function(){
-//        expect(noughtsAndCrossesApp.gameApi)
-//            .to.equal('noughtsAndCrossesApp.service(gameApi,function ($http,$q,gameApiConstants)')
-//    });
-//
-//    it('makeMove function contains appropriate playerNumber and squareNumber', function(){
-//        expect(noughtsAndCrossesApp.gameApi.makeMove)
-//            .to.equal(gameApi.makeMove.playerNumber = 'playerNumber',gameApi.makeMove.squareNumber = 'squareNumber');
-//    });
-//
-//    it('newGame function contains constants', function(){
-//        expect(noughtsAndCrossesApp.gameApi.newGame)
-//            .to.equal(gameApi.newGame.player1 = 'player1Type',gameApi.newGame.player2 = 'player2Type');
-//    });
-//
-//    it('gameApi service contains promise', function(){
-//        expect(noughtsAndCrossesApp.gameApi)
-//            .to.equal(gameApi.callServer);
-//    });
-//
-//});
+describe('gameApiTest', function(){
+
+    var gameApi;
+    var gameModel;
+
+    beforeEach(function(){
+        module('tombola.noughtAndCrosses.services');
+        inject(function($injector){
+            gameApi = $injector.get('gameApi');
+            gameModel = $injector.get('NoughtsAndCrosses/unit-tests/mocks/mock-game-model.js')
+        });
+    });
+
+    it('newGame function should reset the game board', function(){
+        gameApi.newGame();
+        expect(gameModel.gameboard).to.equal('000000000');
```

Comment:
This doesn't make sense - the we're not testing the result of the API-Proxy  - just the effect of the API-Proxy on the game board. 
I think there might be an underlying architectural fault - the API-Proxy  shouldn't update the gameBoard itself: it would be better to inject the proxy into the gameBoard, and have the gameBoard update itself from the API result rather than (as I assume you've done) call the proxy and then have the proxy update the gameboard. 
The idea is to separate the proxy out from the rest of the implementation - we could swap the proxy for something completely different with same input/output and it wouldn't change the rest of the code.